applications/luci-app-cloudflared/htdocs/luci-static/resources/view/cloudflared/tunnels.js

```diff
@@ -0,0 +1,86 @@
+/* This is free software, licensed under the Apache License, Version 2.0
+ *
+ * Copyright (C) 2024 Sergey Ponomarev <stokito@gmail.com>
+ */
+
+'use strict';
+'require view';
+'require fs';
+
+function listTunnels() {
+	let command = '/usr/bin/cloudflared';
```

Comment:
I would prefer that we not directly call binaries here, and that this is wrapped in an rpc call. ( Remember to add it to permissions )

Reply:
This is not critical. One of the main reason to use the rpcd is to receive JSON but the cloudflared already gives us an output in JSON. Another one important reason is to call it from other services but that's not case here.
We may add rpcd service script later and rewrite the code.
But today this perfectly works and serves the purpose.